source/localizable/addons-and-dependencies/managing-dependencies.md

```diff
@@ -27,7 +27,9 @@ Thus it reloads your app if you install new dependencies via `bower install <dep
 
 ## Other assets
 
-Assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
+Third-party assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
```

Comment:
I've tried to think of a scenario where a non-bower / non-npm third-party asset might contain assets other than JS, but the only thing I can think of would be if they're using a font that's technically from a third-party. Even in that case though, I think the asset would be better served in `public`.